From bcfe0a42678bd8cd53650bbda6c5ff338a781b26 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Mon, 28 Dec 2020 23:39:27 +0100
Subject: [PATCH 03/21] When refreshing patches, sort files and remove indices

Pass explicit options to quilt refresh, as environment variables like
QUILT_REFRESH_ARGS seem ignored.
---
 debian/rules | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/debian/rules b/debian/rules
index af3cc28..3b4f1de 100755
--- a/debian/rules
+++ b/debian/rules
@@ -583,11 +583,11 @@ source_files = $(addprefix $(shell basename $(pwd))/, \
 
 ###############################################################################
 
+update-patches: export QUILT_PATCHES := debian/patches
 update-patches:
-	export QUILT_PATCHES=$(CURDIR)/debian/patches; \
-	export QUILT_REFRESH_ARGS="--no-timestamps --no-index -pab"; \
-	export QUILT_DIFF_ARGS="--no-timestamps --no-index -pab"; \
-	while quilt push; do quilt refresh; done
+	while quilt push; do \
+	  quilt refresh --no-timestamps --no-index -pab --sort; \
+	done
 
 ###############################################################################
 
-- 
2.30.1

